homeconnect/api.py

```diff
@@ -66,7 +66,9 @@ def request(self, method: str, path: str, **kwargs) -> Response:
         We don't use the built-in token refresh mechanism of OAuth2 session because
         we want to allow overriding the token refresh logic.
         """
-        url = f"{self.host}/{path}"
+        if not path.startswith("/"):
+            path = "/" + path
+        url = f"{self.host}{path}"
```

Comment:
What about `url = f"{self.host.rstrip('/')}/{path}"`?

Reply:
Good suggestion! I will change it!